### ISSUE TITLE
everytime I do a /join /olm verify etc I have to restart weechat

### ISSUE THREAD

**Issue by r3k2** [poljar/weechat-matrix#71]
sorry for all the QA im doing :) 
when I add a friend this is whats happening:
```
- friends come into channel
- I can type but he wont see it
- I try /olm verify @friend:server.org
- nothing happens I check status buffer, and says can't find it.. I try with * etc same result
- I restart weechat, then I do the /olm verify again, now works and I get the keys
- now I can't type on channel... 
- I need to /reload (sometimes this fixex it) most times I need to yet again restart weechat
```

for /join
```
- I want to join a channel lets say #cyberpunk:hispagatos.org
- I type /join #cyberpunk:hispagatos.org
- I do not see it listed.. nothing on my status buffer
- I restart weechat
- then I can see it .. 
```

**Comment by poljar** [comment]
I believe i reproduced the issue where you can't type on a channel and found a fix. Will push it over the weekend.

Your server seems to dislike keep-alive quite a bit which makes the experience quite unpleasant. I'll check out the other stuff you described later on as well.

**Comment by ReK2Fernandez** [comment]
@poljar any news on this? thanks

**Comment by poljar** [comment]
Sorry forgot to push. I believe the first issue you described should be either fully or partially fixed by c400404e3dd9a1806516348360388813b1efe850. I still need to check if user key queries can be stuck in the same way as the room key sharing.

**Comment by r3k2** [comment]
thanks, we ended up moving to gomuks a couple months a go,  thanks for fixing this.
